CMakeLists.txt

```diff
@@ -79,6 +79,9 @@ INCLUDE_DIRECTORIES(${CMAKE_SOURCE_DIR}/src
 	${CMAKE_BINARY_DIR}/src
 	${CMAKE_BINARY_DIR}/src/shogun)
 
+# detect PYTHON
+FIND_PACKAGE(PythonInterp REQUIRED)
```

Comment:
you should rather fix this problem then in `FindJinja2.cmake`, i.e. put in the beginning of that script that for being able to run it you need `FIND_PACKAGE(PythonInterp REQUIRED)`